web/packages/teleport/src/components/ReAuthenticate/useReAuthenticate.ts

```diff
@@ -14,33 +14,44 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-import React from 'react';
-
 import useAttempt from 'shared/hooks/useAttemptNext';
 
 import cfg from 'teleport/config';
 import auth from 'teleport/services/auth';
 
-export default function useReAuthenticate({
-  onAuthenticated,
-  onClose,
-  actionText = defaultActionText,
-}: Props) {
+import type { MfaAuthnResponse } from 'teleport/services/mfa';
+
+export default function useReAuthenticate(props: Props) {
```

Comment:
A comment about the distinct prompt shapes and how they change the behavior of the hook would be helpful.